cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -1102,4 +1102,9 @@ export default {
     const holdingSection = section.find(Accordion(including(holdingName)));
     cy.expect(holdingSection.find(Badge()).has({ value: `${numOfItems}` }));
   },
+
+  checkAbsenceOfNewMarcBibRecordOption() {
```

Comment:
I'd recommend to move this function to line 346 to have method for similar functionality close to each other. This simplifies search of need actions